### ISSUE TITLE
Telegram crashes if you try to drag the progress bar in the music player

### ISSUE THREAD

**Issue by GneyHabub** [overtake/TelegramSwift#583]
### To reproduce
1. Start playing any music file
2. Click on the progress bar (the one under the name of the file and playback control buttons) and drag it (as if you were trying to skip around the file)

Sometimes the player just freezes and the app crashes after some seconds, sometimes it crashes immediately

### Configuration
**Operating system:** MacBook Air, macOS 11.0.1

**Version of Telegram:** Telegram macOS 7.3 (211334) STABLE

**Used theme**:  Night Accent

### Logs
```
System Integrity Protection: enabled

Crashed Thread:        36  FFMpegMediaFrameSourceContext

Exception Type:        EXC_ARITHMETIC (SIGFPE)
Exception Codes:       EXC_I386_DIV (divide by zero)
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Floating point exception: 8
Termination Reason:    Namespace SIGNAL, Code 0x8
Terminating Process:   exc handler [29252]
```

**Comment by zweeltow** [comment]
Hey! Can you check this bug on latest stable version ([7.4 (212543)](https://t.me/macos_stable_updates_files/82))? 

If this bug is still actual, can you record how it happened please? I can't reproduce it.

**Comment by GneyHabub** [comment]
Yep, the bug is still there: [video](https://youtu.be/ePDIBPAisVc)
What I also noticed is that it doesn't always happen, even with the same file. One time it crashes, another time - it works okay. One more thing that I noticed - it happens far more often with the files that were uploaded by me.

**Comment by LoopKarma** [comment]
@zweeltow bug is reproducible when the audio is long (in my case it's ~ 30 minutes)

> then:
> -  you click fast from the very beginning to the end of file and back again
> - or try to do click with text selection over audio progress bar
> - or right click also leading to crash

**Comment by DaniruKun** [comment]
The bug is still there, just reproduced it on my machine too:
![telegram-cloud-photo-size-4-5789709326707963930-y](https://user-images.githubusercontent.com/5202322/128241451-a5b9e6f6-62df-46d7-a73a-ceac80f979e8.jpg)

**Comment by doctorby** [comment]
For me, it happens mostly with video, can be related to https://github.com/overtake/TelegramSwift/issues/549
Still reproducible on 9.1.238857 Stable

`When the video ends, or you rewind the video by clicking on a specific position on the progress bar -> spiner is shown and video doesn't play anymore, all others videos also don't play, you have to "Force quit" the app and reopen the telegram again. Sometimes in these cases, app hangs and doesn't even respond and you also have to "Force quit".
It happens often and becomes annoying because video content consumption is one of the main parts of using telegram.`
